### PR TITLE
🌐 add Korean translation

### PR DESCRIPTION
**Pull request by sh9351**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "linked",
-  "version": "1.3.1",
+  "version": "1.3.2",
```
Comment (lostdesign) [inline package.json:3]: Can you revert that change as I am usually bumping the version in a tag commit.

**Review by lostdesign — CHANGES_REQUESTED** [review]
So far so good unless the package.json change. Once reverted I will merge right away.

**Comment by sh9351** [comment]
Okay, I reverted the changes

**Comment by lostdesign** [comment]
Alright thanks, sorry for not getting back to you earlier.

**Pull request merged**